### PR TITLE
[FIX]: Show password button has been made.

### LINKED ISSUE #16532: [ENHANCEMENT]: No 'show password' button.
__PROBLEM__:
No **show password** button which makes it very difficult to check the password, resulting in unnecessary requests to server!

### PR DESCRIPTION
**Pull request by ritvikjain99**
It becomes really important to have show password button, so that if user wants to check the typed password he/she can !
Also the button doesn't user lose focus from input. 
Fixes #16532
Now:
ezgif com-video-to-gif(7)

### CODE REVIEW

File: app/theme/client/imports/forms/input.css [hunk 1]
```diff
@@ -5,6 +5,23 @@ textarea.rc-input__element {
 	font-family: inherit;
 }
 
+.rc-input__password {
+	display: flex;
+	align-items: center;
+	border-width: var(--input-border-width);
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected an empty line before property "border-width" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 2]
```diff
@@ -5,6 +5,23 @@ textarea.rc-input__element {
...
+	border-color: var(--input-border-color);
+
+	& > .rc-input__element {
+		width: 90%;
+		border: 0;
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected an empty line before property "border" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 3]
```diff
@@ -5,6 +5,23 @@ textarea.rc-input__element {
...
+	}
+
+	& > .pass-icon {
+		padding: .5em;
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 4]
```diff
@@ -5,6 +5,23 @@ textarea.rc-input__element {
...
+		width: 10%;
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected "width" to come before "padding" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 5]
```diff
@@ -133,45 +150,45 @@ textarea.rc-input__element {
...
 		}
-
+		
 		& .rc-input {
 			&__element {
 				border-color: var(--input-error-color);
 			}
 		}
-	}
+	}
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected newline before "}" (block-closing-brace-newline-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 6]
```diff
@@ -133,45 +150,45 @@ textarea.rc-input__element {
...
-
+		
 		width: 100%;
 		margin: 0.5rem 0;
 		padding: 0.3rem;
-
+		
 		color: var(--input-text-color);
-
+
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 7]
```diff
@@ -149,27 +168,27 @@ textarea.rc-input__element {
...
 	&__wrapper {
 		display: flex;
 		flex-direction: row;
-
+		
 		width: 100%;
 		margin: 0.5rem 0;
 		padding: 0.3rem;
-
+
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

File: app/theme/client/imports/forms/input.css [hunk 8]
```diff
@@ -5,6 +5,25 @@ textarea.rc-input__element {
 	font-family: inherit;
 }
 
+.rc-input__password {
+	display: flex;
+	align-items: center;
+
+	border-width: var(--input-border-width);
```
Comment (sampaiodiego) [inline app/theme/client/imports/forms/input.css]: Codacy Issue found: [Expected an empty line before property "border-width" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4969942)

**Comment by ritvikjain99** [comment]
@gabriellsh please review !

**Comment by josecarlosweb** [comment]
@ritvikjain99 please fix the conflicts.

**Pull request closed without merging**